Sources/VGSCollectSDK/Utils/Loggers/VGSCollectRequestLogger.swift

```diff
@@ -49,7 +49,11 @@ internal class VGSCollectRequestLogger {
 		if let errorData = data {
 			if let bodyErrorText = String(data: errorData, encoding: String.Encoding.utf8) {
 				print("❗Failed ⬇️ VGSCollectSDK response extra info:")
-				print("\(bodyErrorText)")
+				if bodyErrorText.count > maxTextCountToPrintLimit {
+					print("VGSCollectSDK response size is too big to print. Use debugger if needed.")
```

Comment:
should we use log levels instead print here?

Reply:
Actually it is indeed log levels functionality)